pkgs/shells/fish/plugins/default.nix

```diff
@@ -28,6 +28,8 @@ lib.makeScope newScope (self: with self; {
 
   forgit = callPackage ./forgit.nix { };
 
+  fifc = callPackage ./fifc.nix { };
```

Comment:
Could you please sort alphabetically?

Reply:
Of course, thank you for the prompt review.